crates/ruff_python_ast/src/nodes.rs

```diff
@@ -1558,10 +1621,9 @@ impl fmt::Debug for StringLiteralFlags {
 ///
 /// Bytestrings and f-strings are excluded from this enumeration,
 /// as they are represented by different AST nodes.
-#[derive(Debug, Default, Copy, Clone, PartialEq, Eq, Hash)]
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
 pub enum StringLiteralPrefix {
     /// Just a regular string with no prefixes
-    #[default]
     None,
```

Comment:
I think it's a reasonable default as well. But, looking at this enum again, I pondered whether there was actually any real use-case for implementing `Default` on this enum. I can't really think of any situation where it would improve readability to use `StringLiteralPrefix::default()` rather than `StringLiteralPrefix::None`, and I think there is value in forcing people to be explicit here.

There are no places where we use `StringLiteralPrefix::default()` currently; I added it in https://github.com/astral-sh/ruff/pull/10298 more for completeness than anyone else

Reply:
Thank you for the explanation; this reasoning makes sense.